core/domain/question_services.py

```diff
@@ -217,3 +228,266 @@ def update_question(
     updated_question = apply_change_list(question_id, change_list)
     _save_question(
         committer_id, updated_question, change_list, commit_message)
+
+
+def get_new_question_id():
+    """Returns a new question id.
+
+    Returns:
+        str. A new question id.
+    """
+    return question_models.QuestionModel.get_new_id('')
+
+
+def create_question_summary(
+        question_id, creator_id, status):
+    """Creates and stores a summary of the given question.
+
+    Args:
+        question_id: str. ID of the question.
+        creator_id: str. The user ID of the creator of the question.
+        status: str. The status of the question.
+    """
+    question = get_question_by_id(question_id).to_dict()
```

Comment:
Don't change question object to dict. It's better to work with class objects.

Reply:
done